app/views/help/delegated-request.md

```diff
@@ -0,0 +1,5 @@
+<% title "How do I submit a delegated request on behalf of a card holder?" %>
+
+1. If you are making a request on behalf of a card holder, first, make sure you have authority to do so.
+1. When you make the request, include the following note in the Description section: “This request is for [FirstName LastName]’s card.”
+1. Once you submit the request be sure to use the Add Observer feature to notify the card holder that you’ve made this request on their behalf.
```

Comment:
@phirefly Are we going with the term "subscriber" rather than "observer"?

/cc #529